utils/scarb-test-support/src/filesystem.rs

```diff
@@ -23,9 +24,23 @@ pub fn write_simple_hello_script(name: &str, t: &TempDir) {
     );
 }
 
+fn os_path_without_asdf_dir() -> OsString {
+    let os_path = env::var_os("PATH").unwrap();
+
+    if let Ok(asdf_dir) = env::var("ASDF_DATA_DIR") {
```

Comment:
I do not have this env on my machine, but I do have the following:

```bash
ASDF_DIR=/Users/mk/.asdf
```

maybe we can also consider this one?